sonic_installer/bootloader/aboot.py

```diff
@@ -3,6 +3,7 @@
 """
 
 import base64
+from contextlib import contextmanager
```

Comment:
Please organize `from <x> import ...` imports after all `import ...` imports in the appropriate group (without a blank line between), and alphabetize them separately. E.g.,

```
import base64
import collections
import os
import re
import subprocess
import sys
import zipfile
from contextlib import contextmanager
```

Reply:
Will do